### PR TITLE
Create first pull request

### PR DESCRIPTION
**Pull request by yang-yp**
1. 
2.
3.

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```
Comment (darrenlau86) [inline .idea/modules.xml:2]: commnet

File: configuration/environment.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    maxInstances: 3
+  },
+
+  multiCapabilities: [{
+    'browserName': 'chrome',
+    'version': 63,
+    'platform': 'MAC',
+    'count': 1,
+  }, {
+    'browserName': 'firefox',
```
Comment (darrenlau86) [inline configuration/environment.js:23]: test

File: protractor-conf.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  },
+
+  afterLaunch: function (exitCode) {
+    return new Promise(function (resolve) {
+      reporter.afterLaunch(resolve.bind(this, exitCode));
+    });
+  },
+
+  beforeAll: function (done) {
+    return new Promise(function (resolve) {
```
Comment (darrenlau86) [inline protractor-conf.js:79]: testest

**Review by darrenlau86 — CHANGES_REQUESTED** [review]

**Comment by yang-yp** [comment]
@darrenlau86  review again

**Review by darrenlau86 — APPROVED** [review]
approved

**Pull request merged**